src/components/views/messages/MFileBody.tsx

```diff
@@ -143,17 +107,11 @@ export default class MFileBody extends React.Component<IProps, IState> {
         return presentableTextForFile(this.content);
     }
 
-    private downloadFile(fileName: string, text: string) {
+    private downloadFile(fileName: string) {
         this.fileDownloader.download({
```

Comment:
calling `.download` requires the iframe. See FileDownloader::download and the postMessage it needs to send.